__device-tests__/gutenberg-editor-functionality-test-1-visual.test.js

```diff
@@ -204,23 +201,25 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			if ( isAndroid() ) {
 				await blockButton.click();
 			} else {
-				await editorPage.driver.execute( 'mobile: tap', {
-					element: blockButton,
-					x: 10,
-					y: 10,
-				} );
+				await editorPage.driver.executeScript( 'mobile: tap', [
```

Comment:
I realize this was already using a `mobile:` script, but is it necessary? Should we use `click` instead?

Reply:
Nice catch! It was not needed so I've removed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6290/commits/b9856d11f66350d8fd09e85c87615b81e21e4d9e